sql/catalyst/src/main/scala/org/apache/spark/sql/catalog/v2/LookupCatalog.scala

```diff
@@ -137,7 +116,3 @@ trait LookupCatalog extends Logging {
     }
   }
 }
-
-object LookupCatalog {
-  val SESSION_CATALOG_NAME: String = "session"
```

Comment:
it's moved to https://github.com/apache/spark/pull/25368/files#diff-ae651240c4bc2800dd88980479c53cfaR88